jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/AdminResource.java

```diff
@@ -421,7 +422,7 @@ public Boolean apply(@Nullable final String key) {
         tenantOverdueConfigCacheController.remove(tenantRecordId);
 
         // clear tenant-catalog cache by tenantRecordId
-        final CacheController<Long, Catalog> tenantCatalogCacheController = cacheControllerDispatcher.getCacheController(CacheType.TENANT_CATALOG);
+        final CacheController<Long, List<StaticCatalog>> tenantCatalogCacheController = cacheControllerDispatcher.getCacheController(CacheType.TENANT_CATALOG);
```

Comment:
I'm fine with this but isn't it easier to cache an object `VersionedCatalog` instead of an erased list?

Reply:
yes, will fix